index.js

```diff
@@ -80,7 +80,7 @@ const jsonschemaErrors = handler => {
       detail: error.message,
       source: {
         pointer: error.schemaPath.substr(1),
-        parameter: error.dataPath.substr(1)
+        parameter: error.dataPath == undefined ? error.instancePath.substr(1) : error.dataPath.substr(1)
```

Comment:
Can you replace it with this:
```
// ajv v7: dataPath; avj v8: instancePath
parameter: (error.dataPath || error.instancePath).substr(1)`
```

Reply:
Yea! Already changed it!
Sorry, js isn't my main dev language :D